tests/components/tts/test_init.py

```diff
@@ -1226,7 +1230,9 @@ async def test_load_cache_retrieve_without_mem_cache(
 
     req = await client.get(url)
     assert req.status == HTTPStatus.OK
-    assert await req.read() == tts_data
+
+    # File will actually get converted, so it will contain empty ID3 tags
```

Comment:
We shouldn't have empty ID3. The TTS integration has support to write the ID3 tag to the message being read.